core/new-gui/src/app/dashboard/component/feature-container/user-project-section/project-page/ngbd-modal-remove-project-workflow/ngbd-modal-remove-project-workflow.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="modal-header">
+    <h4 class="modal-title" id="modal-basic-title">Remove Workflows From Project</h4>
+    <button (click)="activeModal.close()" aria-label="Close" class="close" type="button">
+        <span aria-hidden="true"> × </span>
+    </button>
+</div>
+
+<div class="modal-body">
+    <table class="table">
+        <thead>
+            <tr>
+                <th><input type="checkbox" [checked]="isAllChecked()" (change)="changeAll()"></th>
```

Comment:
it's better to avoid function call in the html template. it's better to write the statement directly at `isAllChecked()`. this is a minor issue and can be ignored if it is hard to do AND this page is not frequently initialized.

Reply:
will attempt together with code reunification in next PR